### PR TITLE
#155738456 setup development environment

### PR DESCRIPTION
**Pull request by johngorithm**
#### What does this PR do?
- Setup development environment

#### Description of task to be completed    
- setup sonarLinter
- setup findbugs
- setup checkstyle
- setup pmd

#### How should this be manually tested?
- clone the `repo`and run the following on your command-line:
- `./gradlew lint`
- `./gradlew findbugs`
- `./gradlew pmd`
- `./gradlew checkstyle`

#### Screenshots
- SonarLinter
screen shot 2018-11-29 at 1 05 12 am

screen shot 2018-11-29 at 12 28 00 am

- PMD
screen shot 2018-11-29 at 12 28 52 am

- FindBugs
screen shot 2018-11-29 at 12 29 25 am

- CheckStyle
screen shot 2018-11-29 at 12 29 56 am

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#155738456](https://www.pivotaltracker.com/story/show/155738456)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,13 +1,67 @@
...
-/local.properties
-.idea
-/.idea/caches/build_file_checksums.ser
-/.idea/libraries
-/.idea/modules.xml
-/.idea/workspace.xml
 .DS_Store
-/build
-/captures
+
```
Comment (petermwash) [inline .gitignore]: Add the `.DS_Store` into the `.gitignore` file.

**Comment by petermwash** [comment]
@johngorithm  Good job on setting up your development environment. Kindly check on the failing issue on `circleci`.

**Review by petermwash — APPROVED** [review]
LGTM

**Comment by johngorithm** [comment]
Hi @petermwash 
In response to your feedback above, I am yet to set-up `Circle CI build`. I only registered the repo on `circle ci` but no configuration file yet.
That is what I intend to do next

**Pull request merged**